bee-hive/pyproject.toml

```
@@ -5,9 +5,9 @@ description = "A multi-agent platform with the vision to facilitate deploy and r
 authors = ["IBM"]
 license = "Apache 2.0"
 readme = "README.md"
+requires-poetry = ">=2.0"
 packages = [
-    { include = "bee_hive", from = "." },
-    { include = "cli", from = "." }
+	{include = "*", from="bee_hive"}
```

```suggestion
    {include = "*", from="bee_hive"}
```

I removed an embedded TAB

I think the two statements will function the same for bee_hive and cli -- of course additional will get added, which I think is your intent. (I'll test when rebased - and note pyproject.toml is now top level)

I'm not sure 'requires-poetry' is valid, but fine if it works